community/lucene-index/src/main/java/org/neo4j/index/impl/lucene/LuceneCommandApplier.java

```diff
@@ -61,9 +61,19 @@ public boolean visitIndexAddNodeCommand( AddNodeCommand command ) throws IOExcep
         CommitContext context = commitContext( command );
         String key = definitions.getKey( command.getKeyId() );
         Object value = command.getValue();
-        context.ensureWriterInstantiated();
-        context.indexType.addToDocument( context.getDocument( new IdData( command.getEntityId() ), true ).document,
-                key, value );
+
+        // Below is a check for a null value where such a value is ignored. This may look strange, but the
```

Comment:
Maybe extract a method like:
```
private void visitAddCommand( IndexCommand command, EntityId.AbstractData entityId )
{
    Object value = command.getValue();
    if ( value != null )
    {
        String key = definitions.getKey( command.getKeyId() );
        CommitContext context = commitContext( command );
        context.ensureWriterInstantiated();
        context.indexType.addToDocument( context.getDocument( entityId, true ).document, key, value );
    }
}
```
together with this comment to remove a bit of code and comment duplication?